README.md

```diff
@@ -165,36 +165,45 @@ The following command will provide the list of available command line options:
 
     $ python Scout.py --help
 
+You can also use this to get help on a specific provider:
+
+    $ python Scout.py aws --help
+
 For further details, checkout our Wiki pages at <https://github.com/nccgroup/ScoutSuite/wiki>.
 
 After performing a number of API calls, Scout will create a local HTML report and open it in the default browser.
 
+Also note that the command line will try to infer the argument name if possible when receiving partial switch. For
+example, this will work and use the selected profile:
+
+    $python Scout.py aws --pro PROFILE
```

Comment:
`--profile`

Reply:
We can remove that part, but that was exactly what I was trying to show: The default behavior of argparse is to infer the switch if it cannot find it. So in that case, --pro, --prof, --profi, and so on would all work in place of --profile. But if this is confusing, we can remove it and let people figure it out by themselves!

Reply:
Damn, didn't know that! And hadn't read the paragraph above 😅 . So yeah, keep it!